src/App.js

```diff
@@ -732,14 +726,12 @@ class App extends Component {
       name: null,
       user: null
     });
-    if (sessionStorage.getItem('authMode') !== 'external')
-      this.state.keycloak.logout().then(() => {
-        this.setState({
-          keycloak: null
-        });
-        auth.logout();
+    if (sessionStorage.getItem('authMode') !== 'external') {
+      this.authService.logout();
```

Comment:
this is already called. do you need another one?

Reply:
done!